lib/inspec/resources/command.rb

```diff
@@ -32,6 +32,15 @@ def initialize(cmd, options = {})
 
       @command = cmd
 
+      @timeout = options[:timeout]&.to_i || Inspec::Config.cached.final_options['command_timeout']&.to_i
```

Comment:
@clintoncwolfe this is the first time a resource has reached into `Inspec::Config`to look at settings. Any reason not to do this, or any other pattern I could use? Or does this seem reasonable?

Reply:
I don't see any issue with accessing Config from a Resource.